source/tutorials/write-update-documents.txt

```diff
@@ -0,0 +1,291 @@
+         fields </reference/command/insert/#command-fields>` guide in the
+         MongoDB Server manual.  
+
+   * - ``hint (Bson hint)``
+     - | Gets or sets the index to scan for documents. 
+         For more information, see the :manual:`hint statement </reference/command/update/#std-label-update-command-hint>`
+         in the MongoDB Server manual.
+
+   * - ``hint (String hint)``
+     - | Gets or sets the index to scan for documents. 
+         For more information, see the :manual:`hint statement </reference/command/update/#std-label-update-command-hint>`
+         in the MongoDB Server manual.
+
+   * - ``let (Bson variables)``
+     - | A Map of parameter names and values. Values must be constant or closed
+         expressions that don't reference document fields. For more information,
+         see the :manual:`let statement
+         </reference/command/update/#std-label-update-let-syntax>` in the
+         MongoDB Server manual.
+
+   * - ``upsert (Boolean upsert)``
+     - | Specifies whether the update operation performs an upsert operation if no 
+         documents match the query filter. For more information, see the :manual:`upsert
+         statement </reference/command/update/#std-label-update-command-upsert>`
+         in the MongoDB Server manual.
+
+Example
+~~~~~~~
+
+The following code uses the ``updateMany()`` method to find all documents where the 
+``borough`` field has the value ``"Manhattan"``. It then updates the ``borough``
+value in these documents to ``"Manhattan (north)"``. Because the ``upsert`` option is
+set to ``True``, the {+driver-short+} inserts a new document if the query filter doesn't 
+match any existing documents.
+
+
+.. code-block:: java
+   :copyable: true
+
+    ObservableSubscriber <UpdateResult> updateManySubscriber = new SubscriberHelpers.OperationSubscriber<>();
+    restaurants.updateMany(eq("borough", "Manhattan"), set("borough", "Manhattan
+    (north)"), new UpdateOptions().upsert(true)).subscribe(updateManySubscriber);
+    updateManySubscriber.await();
+
+Return Value
+------------
+
+The ``updateOne()`` and ``updateMany()`` methods each return an ``UpdateResult`` 
+object. The ``UpdateResult`` type contains the following properties:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``getMatchedCount()``
+     - | The number of documents that matched the query filter, regardless of
+         how many were updated.
+
+   * - ``getModifiedCount()``
+     - | The number of documents modified by the update operation. If an updated
+         document is identical to the original, it is not included in this
+         count.
+
+   * - ``getUpsertedId()``
+     - | The ID of the document that was upserted in the database, if the driver
+         performed an upsert. Otherwise ``null``.
+         
+   * - ``wasAcknowledged()``
+     - | Returns true if the update was acknowledged.
+
+.. Additional Information
```

Comment:
I: since this header is commented out, the API Documentation section is rendering as a subsection of "Return Value." You could uncomment the "Additional Information" header and maybe add a sentence linking to the [update](https://www.mongodb.com/docs/manual/reference/operator/update/) page in the Server docs